css/styles.css

```diff
@@ -266,3 +266,7 @@ a {
 #audioplayer .plyr__volume--display, #audioplayer .plyr__progress--buffer {
     background-color: #e8e8e8;
 }
+
+#audioplayer .plyr__time--duration {
```

Comment:
If this is truly a firefox only fix, let's only apply the style change in firefox: https://stackoverflow.com/questions/952861/targeting-only-firefox-with-css

Reply:
@shakeelmohamed Done in a108c6d